management/status_checks.py

```diff
@@ -314,6 +314,20 @@ def run_domain_checks(rounded_time, env, output, pool):
 	for domain in sort_domains(ret, env):
 		ret[domain].playback(output)
 
+	run_custom_domain_checks(env, output)
+
+def run_custom_domain_checks(env, output):
```

Comment:
Also I think this would make more sense to users if it is per-domain and included in each domain's section, maybe? (Maybe not for domains but for zones, actually.)